helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -788,9 +792,23 @@ public boolean isReady() {
   }
 
   public String getContent() {
-    return "CallbackHandler{" + "_watchChild=" + _watchChild + ", _preFetchEnabled="
-        + _preFetchEnabled + ", _batchModeEnabled=" + _batchModeEnabled + ", _path='" + _path + '\''
-        + ", _listener=" + _listener + ", _changeType=" + _changeType + ", _manager=" + _manager
-        + ", _zkClient=" + _zkClient + '}';
+    StringBuilder sb = new StringBuilder("iCallbackHandler{_watchChild=");
```

Comment:
extra i here.
(Did you edit all these in vim? :D

Reply:
yes, as i can't adjust font size in IDE and my eyesight is really bad right now (it will become better post july-31st)